pom.xml

```diff
@@ -170,7 +170,9 @@
     <junit.jupiter.version>5.6.2</junit.jupiter.version>
     <junit.vintage.version>5.6.2</junit.vintage.version>
     <kafka.version>2.5.0</kafka.version>
-    <kryo.version>4.0.2</kryo.version>
+    <kryo.version>5.0.3</kryo.version>
+    <kryo4.version>4.0.2</kryo4.version> <!-- old kryo, for spark-client -->
```

Comment:
Add a comment linking to Spark removal JIRA as a TODO?

Reply:
is there a spark removal jira already?

Reply:
sure, that would be HIVE-23885